### PR TITLE
Allow maker to settle a CFD via the UI

### LINKED ISSUE #183: Close CFD
Wiring the frontend/taker with the maker, removing the custom output for closing a cfd (dlc) position.

### PR DESCRIPTION
**Pull request by luckysori**
Fixes #183.

Amounts currently hard-coded.

### CODE REVIEW

File: rust/src/wallet.rs [hunk 1]
```diff
@@ -301,7 +301,39 @@ pub async fn open_cfd(taker_amount: u64, maker_amount: u64) -> Result<()> {
...
+    let channels = channel_manager.list_channels();
+    let channel_details = channels.first().context("No channels found")?;
+    let maker_pk = channel_details.counterparty.node_id;
+
+    let taker_amount_msats = taker_amount * 1000;
+    let maker_amount_msats = maker_amount * 1000;
+
+    let maker_connection_str = format!("{maker_pk}@{MAKER_IP_PORT}");
+    let peer_info = parse_peer_info(maker_connection_str)?;
+    connect_peer_if_necessary(&peer_info, peer_manager.clone()).await?;
```
Comment (bonomat) [inline rust/src/wallet.rs:332]: This shouldn't be necessary. Afaik the node will automatically reconnect on restart if there is an open channel. If there is no open channel to the maker, then we shouldn't be in this function anyway :)

**Review by bonomat — APPROVED** [review]
Looks good

**Review by da-kami — APPROVED** [review]

File: rust/src/wallet.rs (hunk 1 above)
Comment (da-kami) [reply rust/src/wallet.rs:332]: True, we actually found out that it does on my machine and it was a diffferent issue, we could remove this.

Comment (luckysori) [reply rust/src/wallet.rs]: Done with https://github.com/itchysats/10101/pull/207/commits/b961223edf3a7fba28e1368303f11b1be8319a04.

**Review by holzeis — COMMENTED** [review]
ups i've never submitted my review. nothing critical, but submitting anyways.

**Pull request merged**